### DISCUSSION TITLE
房子、食物等必须品，就不能用于期货运作，作为期货便具有了金融属性，必然会出现大问题

### DISCUSSION THREAD

**Discussion by K1vinL** [WeNeedHome/SummaryOfLoanSuspension#842] [Q&A]
就事论事

**Comment by Accelerator6666** [comment]
别说了，当初坚持这一套的专家那时候被骂的狗血淋头。。。其实现房出售之所以成本会高，是因为几方想一起空手套白狼而已，受伤的是买到烂尾楼的老百姓。

**Comment by John-zhan** [comment]
> 就事论事

就该用期房预售制度，不然割韭菜的方式太单一就不好玩了

**Comment by ethanrao** [comment]
这话说的没金融概念，生活必需品都可以进行期货交易，[芝交所](https://www.cmegroup.com/markets/agriculture.html#products)的很多商品都可以进行实物交割。

暴雷怨不到期货运作，主要问题是监管问题，可以期房买卖的国家多了去了，有几个大规模暴雷的。

**Comment by b4o** [comment]
这就是跟随西方政策，不及时跟踪调整的结果，地方主官为了政绩不考虑后代
